.changelog/1975.internal.md

```diff
@@ -0,0 +1 @@
+Use Vite instead of Parcel to build project
```

Comment:
In case something helps, my attempt long ago: https://github.com/oasisprotocol/explorer/compare/master...lw/abandoned/vitest